### ISSUE TITLE
Home app shows information, but cant control Dyson Pure Cool

### ISSUE THREAD

**Issue by wh0di5pl5** [joe-ng/homebridge-dyson-link#26]
I'm running Raspberry PI and everything was working fine until I updated it the other day. I seem to be able to see temprature and information in the home app, but when I try to turn the fan on, it doesn't do anything. The app animates and seems to think it's done something, but the fan is still off... any ideas?

**Comment by azaiszen** [comment]
Anyone?

**Comment by joe-ng** [comment]
can you send the log tagged with [DysonPlatform] along?

**Comment by azaiszen** [comment]
Hey Joe - not too sure how to do that from my Raspberry PI - I found a couple log files, but they appeared to be empty?

**Comment by azaiszen** [comment]
my config is...

`{
	"bridge": {
		"name": "Homebridge",
		"username": "xxxx",
		"port": xxx,
		"pin": "xxx"
	},
	"Description": "JSON API",
 	"accessories": [],
	"platforms": [
		{
		"platform": "BelkinWeMo",
		"name": "WeMo Platform"
		},
		{
	        "platform": "SensiboSky",
	        "name": "Sensibo",
	        "apiKey": "xxxxxxx",
	        "timeLapse": 5,
	        "ai": true,
	        "hideFan": false,
	        "hideHumidity": false,
	        "fixedState": "auto"
		},
		{
      		 "platform": "DysonPlatform",
     		 "name": "DysonPlatform",
     		 "accessories": [
      		{
          	 "ip": "xxxxx",
        	 "displayName": "Dyson",
         	 "serialNumber": "xxxxxxxx",
         	 "password": "xxxxxxx"
   
		}
	]
	}
]
}`

**Comment by joe-ng** [comment]
The log is output to console directly if you run the homebridge from console.

**Comment by azaiszen** [comment]
It’s not bringing up anything for Dyson. Only the other 2 accessories

**Comment by joe-ng** [comment]
just to confirm, not even during the start up of homebridge?

**Comment by azaiszen** [comment]
8-4-22 21:59:10] Loaded plugin: homebridge-camera-ffmpeg
[2018-4-22 21:59:10] Registering platform 'homebridge-camera-ffmpeg.Camera-ffmpeg'
[2018-4-22 21:59:10] ---
[2018-4-22 21:59:10] Loaded plugin: homebridge-camera-ffmpeg-omx
[2018-4-22 21:59:10] Registering platform 'homebridge-camera-ffmpeg-omx.Camera-ffmpeg-omx'
[2018-4-22 21:59:10] ---
[2018-4-22 21:59:10] Loaded plugin: homebridge-dyson-link
homebridge API version: 2.2
[2018-4-22 21:59:10] Registering platform 'homebridge-dyson-link.DysonPlatform'
[2018-4-22 21:59:10] ---
[2018-4-22 21:59:11] Loaded plugin: homebridge-platform-wemo
[2018-4-22 21:59:11] Registering platform 'homebridge-platform-wemo.BelkinWeMo'
[2018-4-22 21:59:11] ---
[2018-4-22 21:59:11] Loaded plugin: homebridge-sensibo-sky
[2018-4-22 21:59:11] Registering platform 'homebridge-sensibo-sky.SensiboSky'
[2018-4-22 21:59:11] ---
[2018-4-22 21:59:11] Loaded config.json with 0 accessories and 3 platforms.
[2018-4-22 21:59:11] ---
[2018-4-22 21:59:11] Loading 3 platforms...
[2018-4-22 21:59:11] [DysonPlatform] Initializing DysonPlatform platform...
[2018-4-22 21:59:11] Loading 0 accessories...
[2018-4-22 21:59:11] [DysonPlatform] Dyson Configure Accessory
[2018-4-22 21:59:11] [DysonPlatform] Finished launching. Start to create accessory from config
[2018-4-22 21:59:12] [WeMo Platform]...

**Comment by joe-ng** [comment]
Did you see any log with [DysonPlatform] after below entry?

[2018-4-22 21:59:11] [DysonPlatform] Finished launching. Start to create accessory from config

**Comment by azaiszen** [comment]
Nope. There is nothing. HomeKit seems to think it’s communicating though, everything animated and I seem to be getting readings for temperature/air quality etc fine. But they may be old too

**Comment by joe-ng** [comment]
the config seems to look fine. maybe we can try to update homebridge version?

**Comment by azaiszen** [comment]
I ran all updates. Unless something new is out in the last couple weeks?

**Comment by joe-ng** [comment]
hmm... just to confirm, you only have one instance of homebridge running for this, right?
i asked because if we didn't see any log after below, it meant it couldn't get the config inside the dyson platform config
[DysonPlatform] Finished launching. Start to create accessory from config

**Comment by azaiszen** [comment]
I’m really new to raspberry pi. I set it all up per the guide online and it was working perfectly. I have it launching homebridge on bootup. To be honest I have no idea how or where it’s running when it starts, but I know it does because everything else is working. Not too sure where all the logs live and all that either. But nothing changed and Dyson was the only thing that randomly stopped working. So I’m stuck for troubleshooting because I don’t know where to start?
